packages/nuxeo-amazon-s3-package/src/main/resources/install/templates/s3binaries/nxserver/config/s3directupload-config.xml.nxftl

```diff
@@ -10,8 +10,8 @@
       <property name="keyValueStore">s3DirectUploadKeyValueStore</property> <!-- not defined there, fallback on default -->
       <targetMaxSizeMB>${nuxeo.s3storage.transient.targetMaxSizeMB}</targetMaxSizeMB>
       <absoluteMaxSizeMB>-1</absoluteMaxSizeMB>
-      <firstLevelTTL>120</firstLevelTTL>
-      <secondLevelTTL>10</secondLevelTTL>
+      <firstLevelTTL>${nuxeo.s3storage.transientstore.ttl}</firstLevelTTL>
+      <secondLevelTTL>${r"${nuxeo.s3storage.transientstore.ttl2:=10}"}</secondLevelTTL>
```

Comment:
What does the ${r""} for?

Also, defaults should be there or in _nuxeo.defaults_, not both imho.

Reply:
`${r".."}` is to not process the expression so it ends up in the XML contribution (we handle them ourselves in XML).
I will move ttl2 to `nuxeo.defaults` then.

Reply:
Moved to `nuxeo.defaults`